eva_sub_cli/submit.py

```diff
@@ -77,9 +98,21 @@ def submit(self, resume=False):
                                               'Authorization': 'Bearer ' + self.auth.token})
             response.raise_for_status()
             response_json = response.json()
-            self.info("Submission ID {} received!!".format(response_json["submissionId"]))
+            self.info(f'Submission ID {response_json["submissionId"]} received!!')
             # update config with submission id and upload url
             self.update_config_with_submission_id_and_upload_url(response_json["submissionId"], response_json["uploadUrl"])
 
         # upload submission
         self._upload_submission()
+
+        # Complete the submission
+        self.verify_submission_dir(self.submission_dir)
```

Comment:
What's the reason to verify again? As it's already been verified before the upload

Reply:
Silly copy/paste